code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -67,6 +67,15 @@
 	///Whether the item icon has a cycling animation
 	var/cycle_anim = TRUE
 
+/obj/item/gun/projectile/shotgun/pump/handle_maptext()
+	var/ammo = length(loaded) || 0
```

Comment:
The 0 should not be necessary, this isn't a lazy list and doesn't seem like it's nulled anywhere except deletion (at which point it's not handlign the maptext anymore)